k8s.gcr.io/images/k8s-staging-apisnoop/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- hh
+- bobymcbobs
```

Comment:
@bobymcbobs would need to be an org member to do reviews/approvals for apisnoop --> https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E

Reply:
Looks like @BobyMCbobs is already an org member.